TurkishFilter/sections/specific.txt

```diff
@@ -299,7 +300,7 @@ sezonlukdizi2.com,sezonlukdizi3.com,sezonlukdizi4.com,sezonlukdizi5.com,sezonluk
 dizipal.pro##.footerreklam
 bloomberght.com##a[href*="/advertpro/"]
 yetkinreport.com###el-4ed034a2 p > a > img
-||cloudflare.com/*/videojs-contrib-ads/*/videojs.ads.js$domain=ozgunkocaeli.com.tr|seskocaeli.com|yenialanya.com
+||cloudflare.com/*/videojs-contrib-ads/*/videojs.ads.js$domain=ozgunkocaeli.com.tr|seskocaeli.com|yenialanya.com|milligazete.com.tr
```

Comment:
Let's make:
||cloudflare.com/*/videojs-contrib-ads/$domain=ozgunkocaeli.com.tr|seskocaeli.com|yenialanya.com|milligazete.com.tr
And there would be no need for a separate rule for CSS.